components/feature/pwa/src/main/java/mozilla/components/feature/pwa/ManifestStorage.kt

```diff
@@ -31,6 +35,47 @@ class ManifestStorage(context: Context) {
         manifestDao.value.getManifest(startUrl)?.manifest
     }
 
+    /**
+     * Load all Web App Manifests with a matching scope for the given URL from disk.
+     * If no manifest is found, an empty list is returned.
+     *
+     * @param url URL of site. Should correspond to an url covered by the scope of a stored manifest.
+     * @param includeTimeout Whether the manifest should only be loaded if the last usage
+     *                       has not passed the defined timeout.
+     * @param currentTime timestamp in milliseconds that is used to validate timeout.
+     */
+    suspend fun loadManifestsByScope(
+        url: String,
+        includeTimeout: Boolean = false,
```

Comment:
I think we can take out the timeout code since its really only needed for install state, which is now done by checking `hasRecentManifest`. We can simplify this function and only take `url` as a parameter.